malcolm/modules/web/parts/websocketserverpart.py

```diff
@@ -2,7 +2,8 @@
 from tornado.websocket import WebSocketHandler, WebSocketError
 
 from malcolm.core import Part, json_decode, deserialize_object, Request, \
-    json_encode, Subscribe, Unsubscribe, Delta, Update, Error, UnexpectedError
+    json_encode, Subscribe, Unsubscribe, Delta, Update, Error, Response
+from malcolm.core.request import PathRequest
```

Comment:
Can you put PathRequest and FieldError in malcolm.core.__init__ and import from there please